ocular/src/chain/registry.rs

```diff
@@ -68,7 +66,7 @@ pub async fn get_chain(name: &str) -> Result<ChainInfo, ChainRegistryError> {
 async fn get_content(path: String) -> Result<reqwest::Response, ChainRegistryError> {
     octocrab::instance()
         .repos("cosmos", "chain-registry")
-        .raw_file("HEAD".to_string(), path)
+        .raw_file("88bde7fb534ed6f7c26c2073f57ec5135b470f56".to_string(), path)
```

Comment:
We previously talked about testing every chain once we pin the registry commit (https://github.com/PeggyJV/ocular/issues/28), would we wanna loop that into this PR or defer it to a different one?

Reply:
Seems like a good thing to add to CI in another branch. Have it run on releases?